### PR TITLE
Remove unnecessary dashboard queries

### PR DESCRIPTION
**Pull request by yenienserrano**
### Description
Removed unnecessary requests in dashboard views

### Issues Resolved
- #5237 

### Evidence
<details><summary>Screenshot</summary>

Security events

image

FIM

image

Office 365

image

Amazon AWS

image

Google Cloud Plataform

image

GitHub

image

Policy monitoring

image

System auditing

image

OpenSCAP

image

CIS-CAT

image

VirusTotal

image

Osquery

image

Docker listener

image

MITRE ATT&CK

image

PCI DSS

image

GDPR

image

HIPAA

image

NIST

image

TSC

image

Deploy agent

image

</details> 

### Test

Verify that all dashboards continue to function as required and no request is made.

Verify that when you are going to deploy an agent, the necessary requests are made.

### Check List
- [x] All tests pass
  - [x] `yarn test:jest`
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [x] Update [CHANGELOG.md](./../CHANGELOG.md)
- [x] Commits are signed per the DCO using --signoff

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -37,6 +37,10 @@ All notable changes to the Wazuh app project will be documented in this file.
 - Fixed TypeError in FIM Inventory using new error handler [#5364](https://github.com/wazuh/wazuh-kibana-app/pull/5364)
 - Fixed error when using invalid group configuration [#5423](https://github.com/wazuh/wazuh-kibana-app/pull/5423)
 
+### Removed
+
+- Removed unnecessary dashboard queries caused by the deploy agent view. [#5453](https://github.com/wazuh/wazuh-kibana-app/pull/5453)
+
```
Comment (Desvelao) [inline CHANGELOG.md:43]: **nitpick**: this will have to be updated to the target release.

File: public/controllers/overview/overview.js [hunk 2]
```diff
@@ -283,7 +283,7 @@ export class OverviewController {
         timefilter.setRefreshInterval(this.commonData.getRefreshInterval());
       }
 
-      if (typeof this.agentsCount === 'undefined') {
+      if (typeof this.agentsCount === 'undefined' && newTab === 'welcome') {
```
Comment (Desvelao) [inline public/controllers/overview/overview.js:286]: **question**: I don't know what is the reason for this change. Could you elaborate?

Comment (yenienserrano) [reply public/controllers/overview/overview.js:286]: The request is restricted to the welcome view, since it is the only view where these values are needed.

**Comment by Desvelao** [comment]
# Wazuh dashboard

Legend:
:black_circle:: none
:green_circle:: pass
:yellow_circle:: warning
:red_circle:: fail
:white_circle:: not applicable

## UI

| Test | Chrome | Firefox | Safari |
| --- |  --- | --- | --- |
| Verify that all dashboards continue to function as required and no request is made. | :green_circle: | :black_circle: | :black_circle: |
| Verify that when you are going to deploy an agent, the necessary requests are made. | :green_circle: | :black_circle: | :black_circle: |

**Details**
<details>
<summary>:green_circle: Verify that all dashboards continue to function as required and no request is made.</summary>

Chrome - :green_circle:
image

Firefox - :black_circle:

Safari - :black_circle:

</details>

<details>
<summary>:green_circle: Verify that when you are going to deploy an agent, the necessary requests are made.</summary>

Chrome - :green_circle:
image

Firefox - :black_circle:

Safari - :black_circle:

</details>

**Review by Desvelao — APPROVED** [review]
**preview**

code :green_circle: 
test : :green_circle: https://github.com/wazuh/wazuh-kibana-app/pull/5453#issuecomment-1566903244
